db/types.go

```diff
@@ -180,3 +182,72 @@ type QueryEventsResponse struct {
 	Pagination PageResponse `json:"pagination"`
 	Events     []NftEvent   `json:"events"`
 }
+
+type QueryRankingRequest struct {
+	StakeholderId   string   `form:"stakeholder_id"`
+	StakeholderName string   `form:"stakeholder_name"`
+	Creator         string   `form:"creator"`
+	Type            string   `form:"type"`
+	Collector       string   `form:"collector"`
+	After           int64    `form:"after"`
+	Before          int64    `form:"before"`
+	IncludeOwner    bool     `form:"include_owner"`
+	IgnoreList      []string `form:"ignore_list"`
```

Comment:
On second thought, would a feature that filter addresses that own more than a threshold (say, 100) be also useful?

Reply:
Own 1 NFT class more than a threshold or own any NFT more than a threshold? I don't really understand the use cases.

Reply:
own 1 nft class more than a thersold, just as a extreme value filter
one would expect the owner or creator or api wallet would own most of the api minted, by filtering them away with a thersold it might be useful for analysis